spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'with Authenticated User'
+
+  describe 'GET #index' do
+    context 'when fetching all the books' do
+      let!(:books) { create_list(:book, 3) }
```

Comment:
let is `lazy loading` this means that, at the moment you call the variable, the books objects array is created, so we don't need to add the bang (!). Please remove it.

Reply:
Done